NewForce-at-Mountwest/bangazon-site-the-grave-trolls
# Aw local delivery

# Description

When creating a new product to sell, a user can specify if the product is available for local delivery. This is done by checking a checkbox affordance. If the user enters a city name during the product creation process, the user can then search by city name in the search bar.

## Type of change

Please delete options that are not relevant.


# Testing Instructions

- Add/commit your current branch
- Checkout aw-local-delivery branch
- In Package Manager Console, run add-migration isLocalAdd
- After that is complete, run update-database (this adds the isLocal bool to the Product table in the database)
- Adjust the appsettings.json to match your local settings
- Run the application
- Click 'Sell Something' on the nav bar
- Create a new product. Be sure to check the 'Available for local delivery' checkbox and enter a City name
- Once the product is created, the next screen is its Details view. Check to see if the 'Local delivery' option is checked
- Search the city using the search bar in the nav bar
- Results should appear for only products in that city, whether the product has been labeled as local or not



# Checklist:
